main.tf

```diff
@@ -242,13 +260,24 @@ resource "kubernetes_config_map" "config_map" {
   }
 }
 
+resource "kubernetes_config_map" "reload_pod" {
+  metadata {
+    name = "${local.app_name}-${random_pet.reload_pod.id}"
+  }
+
+  data = {
+    "random-pet" = "${local.app_name}-${random_pet.reload_pod.id}"
+  }
+}
+
 resource "kubernetes_secret" "secret" {
   metadata {
     name = local.app_name
   }
 
   data = {
-    "MYSQL"       = var.database_connection_string
+    # "MYSQL"       = var.database_connection_string
+    "MYSQL"       = local.db_conn_string
```

Comment:
This isn't a good idea.  The connection string is sensitive and shouldn't be stored in a config map.  A better method here would be to just pass through the version of the terraform module or something that will always increment as a regular env var to the deployment.